### PR TITLE
Rel `v3.1.0` Prep

### PR DESCRIPTION
**Pull request by aclark02-arcus**
Prepping for a new minor release

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,34 +1,40 @@
-# riskassessment (development version)
+# riskassessment 3.1.0
+
+### User Enhancements
+* Allow users to specify decisions within the CSV file used to upload pkgs (#663)
+* Drastically shorten waiting time in `Code explorer`/ `Package explorer` by reading tarballs in memory (#707)
```
Comment (jthompson-arcus) [inline NEWS.md:5]:
```suggestion
* Drastically shortened waiting time in `Code explorer`/ `Package explorer` by reading tarballs in memory (#707)
```

File: NEWS.md [hunk 2]
```diff
@@ -1,34 +1,40 @@
...
-* Updated 'About' page to highlight individual contributors on the project, leveraging `bslib` cards.
-* Allow users to specify decisions in the CSV upload file (#663)
-* Fix bug causing application to crash when trying to delete zero packages (#781)
-* Add `Decision` column to Package Dependencies tab's table output (#774)
-* Add `Decision Summary` card to Package Dependencies tab (#774)
-* Attach user name to batch upload decisions (#785)
+
+### For Devs
+* Added non-`shinymanager` deployment option so that users may use their Posit Credentials instead (#700)
+* Move package upload process to a function, allowing devs to populate the database outside of the application
```
Comment (jthompson-arcus) [inline NEWS.md:30]:
```suggestion
* Moved package upload process to a function, allowing devs to populate the database outside of the application
```

File: NEWS.md [hunk 3]
```diff
@@ -1,34 +1,40 @@
...
+* Added tests for the `Code Explorer` and `Function explorer` modules
+* Added repo specification to configuration file (#701)
+* Ensured configuration checkers run when configuring the database
+* Utilize `{archive}` to parse DESCRIPTION file, increasing speed in package upload process
```
Comment (jthompson-arcus) [inline NEWS.md:34]:
```suggestion
* Utilized `{archive}` to parse DESCRIPTION file, increasing speed in package upload process
```

Comment (jthompson-arcus) [reply NEWS.md:34]: This also seems more like a user enhancement to me.

File: NEWS.md [hunk 4]
```diff
@@ -1,34 +1,40 @@
...
+* updated manifest file and deployments article
```
Comment (jthompson-arcus) [inline NEWS.md:35]:
```suggestion
* Updated manifest file and deployments article
```

**Review by jthompson-arcus — CHANGES_REQUESTED** [review]
In addition to the notes below. I noticed that we have no mention of updating the R version and our PPM snapshot date. Should probably add a note about that under the "For Devs" section.

**Comment by aclark02-arcus** [comment]
@jthompson-arcus, I made the changes you suggested. I just requesting your review to re-trigger the manifest build. Will merge when that commit is added.

**Pull request merged**